src/common/simple_spin.cc

```diff
@@ -18,9 +18,12 @@
 #include <stdint.h>
 #include <pthread.h>
 
-static uint32_t bar = 13;
-static uint32_t *foo = &bar;
- 
+#if (__arm__ || __aarch64__)
```

Comment:
Modelled on our existing code in src/test/test_arch.cc should it be changed as well?